src/claims/models/investor-uniqueness.model.ts

```diff
@@ -0,0 +1,24 @@
+import { ApiProperty, OmitType, PartialType } from '@nestjs/swagger';
+import {
+  ClaimType,
+  InvestorUniquenessClaim,
+  ScopeType,
+} from '@polymeshassociation/polymesh-sdk/types';
+
+import { ClaimModel } from '~/claims/models/claim.model';
+
+export class InvestorUniquenessModel extends PartialType(OmitType(ClaimModel, ['claim'] as const)) {
+  @ApiProperty({
+    description: 'Details of the Claim containing type and scope',
+    example: {
+      type: ClaimType.InvestorUniqueness,
+      scope: {
+        type: ScopeType.Identity,
+        value: '0x0600000000000000000000000000000000000000000000000000000000000000',
+      },
+      cddId: '0x0600000000000000000000000000000000000000000000000000000000000000',
+      scopeId: '0x0600000000000000000000000000000000000000000000000000000000000000',
+    },
+  })
+  readonly claim: InvestorUniquenessClaim;
```

Comment:
I know its a bit redundant, but you need to create a corresponding model for `InvestorUniquenessClaim` and you that here. That we we get to describe what each field means. We avoid using interface directly from SDK as return param (FYI there are a few cases left from the past where we have done that but we should avoid it)